app/helpers/application_helper/toolbar/x_vm_center.rb

```diff
@@ -310,7 +310,7 @@ class ApplicationHelper::Toolbar::XVmCenter < ApplicationHelper::Toolbar::Basic
           :confirm   => N_("The selected snapshot will be permanently deleted. Are you sure you want to delete the selected snapshot?"),
           :url_parms => "main_div",
           :onwhen    => "1",
-          :klass     => ApplicationHelper::Button::GenericFeatureButtonWithDisable,
+          :klass     => ApplicationHelper::Button::VmSnapshotRemoveOne,
           :options   => {:feature => :remove_snapshot}),
```

Comment:
@romanblanco Is `:options` here needed when we use a specific button class rather than `GenericFeatureButtonWithDisable` ?